src/styles.scss

```diff
@@ -641,7 +641,7 @@ $green-font-color: #19B028;
 .global__top-bar {
   position: fixed;
   background: var(--background);
-  z-index: 30;
+  // z-index: 30; Should not float above anything like emoji picker
```

Comment:
this breaks a lot of styling - try scrolling down on the "trends" page with this branch checked out. It looks something like this:
![image](https://user-images.githubusercontent.com/81658138/122686802-073c7880-d1c8-11eb-8e66-943b99e4851c.png)

Reply:
instead let's add styling to `.emoji-picker__wrapper` and set it's z-index to be higher than z-index for `.global__top-bar`